blingfire/src/lib.rs

```diff
@@ -104,7 +75,9 @@ where
 
 #[cfg(test)]
 mod tests {
-    use crate::{text_to_sentences, text_to_words};
+    use super::{text_to_sentences, text_to_words, MAX_SOURCE_LENGTH};
```

Comment:
why change to a relative import?

Reply:
`MAX_SOURCE_LENGTH` isn't exported from the crate (and it shouldn't be). In general I think _unit_ tests should import from the parent module, since they're tests for that module, not for the whole crate. Integration tests (define in `tests/`) should use the crate exports.